docs/specifications/tests/Methods.md

```diff
@@ -1,69 +1,632 @@
-# Methods common to Test Specifications
+# Methods common to Test Case Specifications
 
-This is a list of generic methods used in many test specifications. The
-test cases that makes use of any of these methods should link directly to
+This is a list of generic methods used in many Test Case specifications. The
+Test Cases that makes use of any of these methods refer directly to
 this text.
 
-### Method 1: Obtain the parent domain
+## Method 1: Parent zone
 
-This metod tries to obain the parent domain from the domain used as input
-to this method.
+The method has been removed. The function is integrated in [BASIC01] instead.
 
-1. A recursive SOA-record lookup for the child domain name starting at the
-   root domain should be done, and the steps of the process recorded.
-2. If the recursion reaches a name server that responds with a redirect
-   directly to the requested domain, including functional glue, the test
-   succeeds. The domain through which the name server was found is
-   considered the parent domain.  
-3. If the recursion reaches a name server that authoritatively responds
-   with NXDOMAIN for the child domain, the test succeeds. The domain through
-   which the name server was found is considered the parent domain.
+-------------------------------------------------------------
 
-### Method 2: Obtain "glue Name records" from parent
+## Method: Get delegation NS names and IP addresses
 
-This method tries to obtain the authoritative name servers from the
-delegation of the parent domain.
+### Method identifier
+**get-del-ns-names-and-ips**
 
-1. Obtain the parent domain as input from Method 1.
-2. Send a query to the parent domain asking for the list of name servers
-   authoritative for the domain that is being tested 
-3. Record the list of name servers obtained from the authority section 
+### Objective
 
-### Method 3: Obtain name servers from child
+Obtain the name server names (from the NS records) and the IP addresses (from 
+glue records) from from the delegation of the given zone (child zone) from 
+the parent zone. Glue records are address records for [in-bailiwick] name 
+server names, if any. Obtain the IP addresses for the [out-of-bailiwick] 
```

Comment:
It is unclear to me what this method does with glue records that are [out-of-bailiwick]. As written my reading is that it returns glue address records for [in-bailiwick] NS and resolves [out-of-bailwick] NS. Same applies to other

Reply:
Here Zonemaster wants to get all addresses that are supposed to point at name servers that are authoritative for the zone. Both sources are meant to be used. I will clarify. Thank you for the comment.